.github/workflows/rush.yml

```
@@ -0,0 +1,33 @@
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+      - name: Git config user
+        uses: snow-actions/git-config-user@v1.0.0
+        with:
+          name: "Rush Workflow CI"
+          email: "190986\+drazisil@users.noreply.github.com"
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 19: found unknown escape character

(syntax-check)